LBHackney-IT/lbh-fss-portal-frontend
# Add login tests

**Description**

Add front-end validation for the login component, along with corresponding tests 

_Note - styling is still outstanding_ 

**Test instructions**

- pull this branch 
- navigate to the login page (path /) but can be accessed via the 'logout' button if already logged in 
- enter dummy values into the email and password fields to check validation criteria is working as expected 
  - email validation criteria: password must be less than 255 characters, and following email address pattern  
  - password validation criteria: password must be less than 255 characters, more than 6 characters, contain at least one capital letter, contain at least one number 
- run tests using `yarn test` and check tests pass **note** - empty test files have been created for all components, therefore if running all tests you will see multiple failed tests, this merge request just contains tests for the Login component, therefore please run tests for the Login component only (i.e. press 'p' to filter tests by filename, and enter 'Login.test') you should then see 11 tests passed (see screenshot below)

![image](https://user-images.githubusercontent.com/69238887/90781016-6a365800-e2f6-11ea-9896-8f29371f8efe.png)

**ClickUp Ticket**

https://app.clickup.com/t/9e5u6t
